assets/scss/components/_toggle-button.scss

```diff
@@ -2,28 +2,59 @@
 Toggle Button
 
 Markup:
-<span class="toggle-btn {{modifier_class}}">Toggle Button</span>
```

Comment:
As far Im aware of, @stingrayfunk created toggle button component for API subscription.